webrender/src/util.rs

```diff
@@ -150,6 +150,15 @@ pub fn subtract_rect<U>(rect: &TypedRect<f32, U>,
         }
     }
 }
+
+pub fn ensure_float_is_normal(x: f32) -> f32 {
```

Comment:
This is rather minor, since the function is only used once, but if we want to make it generally useful, I think we should have an API like `get_normal(x: f32) -> Option<f32>`, so that the function clients do `unwrap_or(0.0)` (or other value, depending on the context).